src/ContactForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+class ContactForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      name: '',
+      avatar: '',
+      occupation: '',
+    };
+  }
+
+  handleNameChange(event) {
```

Comment:
See if you can reduce all three of these `handle` functions into one like we included in the text book.